.github/workflows/backwards-compatibility.yml

```diff
@@ -0,0 +1,58 @@
+name: "Check Backwards Compatibility"
+
+on:
+  pull_request:
+  push:
+
+jobs:
+  coding-standards:
+    name: "Check Backwards Compatibility"
+
+    runs-on: ${{ matrix.operating-system }}
+
+    strategy:
+      matrix:
+        dependencies:
+          - "locked"
+        php-version:
+          - "7.4"
+        operating-system:
+          - "ubuntu-latest"
+
+    steps:
+      - name: "Checkout"
+        uses: "actions/checkout@v2"
+        with:
+          fetch-depth: 0
+
+      - name: "Install PHP"
+        uses: "shivammathur/setup-php@v2"
+        with:
+          coverage: "pcov"
```

Comment:
I don't think there is any need for coverage tool here: See https://github.com/marketplace/actions/setup-php-action#disable-coverage

Similar comment applies to all the other workflows apart from the test one.

Reply:
I kept this to keep the files as similar as possible, allowing for easy diffing between each other.

Until github actions allow imports of external configurations, it makes sense to keep them close to ease maintenance.